src/doomstat.h

```diff
@@ -69,6 +69,7 @@ extern char *MAPNAME(int e, int m);
 extern  boolean modifiedgame;
 
 extern boolean have_ssg;
+#define ALLOW_SSG (gamemode == commercial || STRICTMODE(have_ssg))
```

Comment:
Isn't this rather "critical" than "strict"?

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1920/commits/edf461cbacdbdf85b7565f497982240aab670bee